ext/build_defaults.yaml

```diff
@@ -100,6 +100,11 @@ platform_repos:
     repo_location: repos/solaris/11/**/*.i386.p5p
   - name: solaris-11-sparc
     repo_location: repos/solaris/11/**/*.sparc.p5p
+  - name: windows
+    repo_location: repos/windows
+  - name: windowsfips
+    repo_location: repos/windowsfips
```

Comment:
Suggest restricting `repo_location: repos/windows/*msi` and same for `windowsfips` so that we don't include the `wixpdb` files, for example, here's the complete directory for 7.24.0: https://builds.delivery.puppetlabs.net/puppet-agent/7.24.0/repos/windows/

I think we also want to exclude the latest symlinks, otherwise we'll end up with two copies of the 32 and 64-bit MSI. I'm not sure how to do that as a file pattern?